chrome/content/zotero/integration/quickFormat.js

```diff
@@ -1437,6 +1433,7 @@ var Zotero_QuickFormat = new function () {
 	 */
 	function _onProgress(percent) {
 		var meter = document.querySelector(".citation-dialog.progress-meter");
+		meter.style.visibility = 'visible';
```

Comment:
Why not in CSS or HTML style attribute?

Reply:
Totally fair - I moved this logic into the stylesheet